### PR TITLE
hpc: Add slurm sanity test

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

File: tests/hpc/slurm_master.pm [hunk 1]
```diff
@@ -12,27 +12,50 @@
...
 
-    # setup slurm master
-    script_run "groupadd -r slurm";
-    script_run "useradd -r -g slurm -d /var/run/slurm -s /bin/false -c \"SLURM Workload Manager\" slurm";
-    assert_script_run 'sed -i "/^ControlMachine.*/c\ControlMachine=slurm-master" /etc/slurm/slurm.conf';
-    script_run "cat /etc/slurm/slurm.conf | grep Control";
+    # install slurm
+    zypper_call('in slurm-munge');
+
+
```
Comment (okurz) [inline tests/hpc/slurm_master.pm]: delete double blank lines

**Comment by Soulofdestiny** [comment]
updated,
verification runs:
master: http://drnek.arch.suse.de/tests/540
slave: http://drnek.arch.suse.de/tests/541

@asmorodskyi @okurz please review again

**Comment by asmorodskyi** [comment]
LGTM

**Comment by Soulofdestiny** [comment]
If you have any concerns about code duplication: https://progress.opensuse.org/issues/20112

**Pull request merged**